client/packages/reports/src/ListView/ListView.tsx

```diff
@@ -63,6 +64,15 @@ export const ListView = () => {
     },
     [navigate]
   );
+
+  if (
+    !store?.preferences?.omProgramModule ||
+    !stockAndItemReports?.length ||
+    !expiringReports?.length
```

Comment:
This fails if *any* of the conditions are `true`, so I'm getting the "Nothing here" even when I have reports enabled.

I think they should probably be `&&`, right?

And maybe doesn't need `!store?.preferences?.omProgramModule`? Would you ever hide the reports even though they are available?

Reply:
I guess not since the script for program pre-configs the reports too...